cli/bin/test-binary.sh

```diff
@@ -5,10 +5,11 @@ BASEDIR=`dirname $0`/..
 cd $BASEDIR
 
 PATH=$(pwd)/dist:$PATH
-cp tests/data/dcos.toml $DCOS_CONFIG
 if [ -f "$BASEDIR/env/bin/activate" ]; then
+        cp tests/data/dcos.toml $DCOS_CONFIG
 	source $BASEDIR/env/bin/activate
 else
+        export DCOS_CONFIG=tests/data/dcos.toml
```

Comment:
it looks that way in this view, but in this view it looks fine: https://github.com/dcos/dcos-cli/blob/17bb05ae14ed2d568a81f6c17a58b79d567c0ebf/cli/bin/test-binary.sh